diff --git a/Tools/ardupilotwaf/git_submodule.py b/Tools/ardupilotwaf/git_submodule.py
index be28216e9c..ee46c11185 100644
--- a/Tools/ardupilotwaf/git_submodule.py
+++ b/Tools/ardupilotwaf/git_submodule.py
@@ -69,11 +69,12 @@ class update_submodule(Task.Task):
         return out.strip() == head

     def runnable_status(self):
+        self.non_fast_forward = []
+        return Task.SKIP_ME
         e = self.env.get_flat
         cmd = e('GIT'), 'submodule', 'status', '--recursive', '--', e('SUBMODULE_PATH')
         out = self.generator.bld.cmd_and_log(cmd, quiet=Context.BOTH, cwd=self.cwd)

-        self.non_fast_forward = []

         # git submodule status uses a blank prefix for submodules that are up
         # to date
